jwst/assign_wcs/tests/test_wcs.py

```diff
@@ -189,6 +192,10 @@ def test_create_fitswcs(tmpdir, create_model_3d):
     assert_allclose((ra, dec), (gra, gdec))
 
 
+@pytest.mark.xfail(
+    Version('0.18.1') < Version(gwcs.__version__) < Version('0.18.2'),
```

Comment:
Maybe I don't understand the subtleties python arithmetic used in this case, but how can a value be both greater than 0.18.1 and less than 0.18.2 at the same time? Should one of those entries perhaps have an equals sign added to it, e.g. `0.18.1 <=`?

Reply:
gwcs 0.18.2 is not released yet, and the current dev version is 0.18.2a1.dev18+ge638714.

```python
>>> from packaging.version import Version
>>> Version('0.18.1') < Version("0.18.2a1.dev18+ge638714") < Version('0.18.2')
True
```

Dev versions are in between patch versions and always start with the unreleased version.  So 0.18.2dev < 0.18.2.

Reply:
Ah ha!  Learn something new every day. See, I told you it was probably because I didn't completely understand the math involved. So ... never mind!